	public void testIgnoreError() throws Exception {
		String str = "<?php @$a->foo(); ?>";
		initialize(str);

		List<IgnoreError> ignoreErrors = getAllOfType(program,
				IgnoreError.class);
		Assert.assertTrue("Unexpected list size.", ignoreErrors.size() == 1);
		ignoreErrors.get(0).setExpression(
				ast.newFunctionInvocation(
						ast.newFunctionName(ast.newScalar("bar")), null));
		rewrite();
		checkResult("<?php @bar(); ?>");
	}

